### DISCUSSION TITLE
Tailwind Doc

### DISCUSSION THREAD

**Discussion by aodamuz** [alpinejs/alpine#3167] [1. Help]
## Hello community!
I would like to know if it is possible to run the Tailwind documentation locally since I have been at my grandparents' house for 15 days and from here I cannot connect to the Internet. I need to work while taking care of my grandparents, but I haven't been able to run the documentation locally. I have cloned the repository and have the Markdown files, but I don't see a script that will allow me to run the documentation.

**Comment by ekwoka** [comment]
Well, for starters, this is the Alpine repo.

The documentation site is a different private repo.

Is reading the markdown not enough though?

Like, that's all the information there is.
